test-suite.html

```diff
@@ -93,8 +93,8 @@ <h2 id="writing-tests-writing-your-test">Writing your test</h2>
 
 	<p>Your file is built up of two or three sections, separated by ten or more dashes <code>-</code>, starting at the begin of the line:</p>
 	<ol>
-		<li>Your language snippet. The code you want to compile using Prism. (<strong>required</strong>)</li>
-		<li>The simplified token stream you expect. Needs to be valid JSON. (<strong>required</strong>)</li>
+		<li>Your language snippet. The code you want to tokenize using Prism. (<strong>required</strong>)</li>
+		<li>The simplified token stream you expect. Needs to be valid JSON. (<strong>required</strong> or <em>optional</em>)</li>
```

Comment:
This should probs just say "optional" (since it can't really be both), and explain the behavior when it's not present.